layers/opentelemetry/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# https://github.com/brefphp/extra-php-extensions/blob/master/layers/yaml/Dockerfile
+
+ARG PHP_VERSION
+ARG BREF_VERSION
+FROM bref/build-php-$PHP_VERSION:$BREF_VERSION AS ext
+
+RUN MAKEFLAGS="-j $(nproc)" && pecl install opentelemetry > /dev/null
```

Comment:
Can we install a specific version here: `pecl install opentelemetry-1.0.0`. I know many of the other extensions here don't do that, but that is tech debt.